pkgs/applications/misc/rtfm/default.nix

```diff
@@ -0,0 +1,76 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchurl
+, callPackage
+, crystal_1_8
+, libadwaita
+, wrapGAppsHook4
+, gobject-introspection
+, desktopToDarwinBundle
+, webkitgtk_6_0
+, sqlite
+, gi-crystal
+}:
+let
+  gtk-docs = callPackage ./gtk-docs.nix { };
```

Comment:
Should we passthru this?

Reply:
Like this?
```nix
let
  passthru = {
    gtk-docs = callPackage ./gtk-docs.nix { };
  };
in
```

Reply:
normally we inherit into passthru so that overriding passthru is not changing things